sauce/features/budget/days-of-buffering/index.js

```diff
@@ -0,0 +1,44 @@
+import { Feature } from 'core/feature';
+import { getEntityManager } from '../../../helpers/toolkit';
+
+import outflowTransactionsFilter from './outflowTransactionsFilter';
+import generateReport from './generateReport';
+import { render, shouldRender } from './render';
+
+export class DaysOfBuffering extends Feature {
+  injectCSS() { return require('./index.css'); }
+
+  constructor() {
+    super();
+    this.historyLookup = parseInt(ynabToolKit.options.DaysOfBufferingHistoryLookup);
+    this.transactionFilter = outflowTransactionsFilter(this.historyLookup);
+    this.lastRenderTime = 0;
+    this.observe = this.invoke;
+  }
+
+  invoke() {
+    if (!shouldRender(this.lastRenderTime)) return;
```

Comment:
We could move this check into `shouldInvoke` as that's what it's effectively accomplishing, yeah? It's not something that would stop a merge, just would make more sense to me.